nncf/torch/quantization/algo.py

```diff
@@ -347,7 +349,8 @@ def __init__(self, quant_config: Dict, target_model: NNCFNetwork,
                          precision_init_params, range_init_params,
                          hw_config)
 
-        self._pattern_fusing_graph = PT_HW_FUSED_PATTERNS.get_full_pattern_graph()
+        self._pattern_fusing_graph =\
+            PatternsManager().get_full_pattern_graph(BackendType.TORCH, TargetDevice.ANY)
```

Comment:
Please, use the predefined `TargetDevice`, since the quantization is possible not only for `ANY` device.

Reply:
Please see https://github.com/openvinotoolkit/nncf/pull/1566/files/01b9ab9f9143ec0361290f5bb322521e4bdf0f83#r1105622381